WalletWasabi.Fluent/ViewModels/AddWallet/AddedWalletPageViewModel.cs

```diff
@@ -44,4 +45,24 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 
 		_wallet = UiContext.WalletRepository.SaveWallet(_walletSettings);
 	}
+
+	private async Task AutoLoginAsync(WalletCreationOptions? options)
+	{
+		var password =
+			options switch
+			{
+				WalletCreationOptions.AddNewWallet add => add.Password,
+				WalletCreationOptions.RecoverWallet rec => rec.Password,
```

Comment:
Auto login HW wallets too, the password is always empty.

```suggestion
				WalletCreationOptions.RecoverWallet rec => rec.Password,
				WalletCreationOptions.ConnectToHardwareWallet => "",
```